app/models/study.rb

```diff
@@ -10,6 +10,11 @@ def self.current_interventional
     self.interventional and self.current
   end
 
+  def self.try(nct_id='NCT00023673')
```

Comment:
`try` is a method in the `ActiveSupport` module, so this method will overwrite it. What is the purpose of this method?